index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html class="box-border">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./input.css">
+    <title>How to play</title>
+</head>
+<body class="flex flex-col m-auto bg-no-repeat bg-cover bg-fixed bg-[url('/img/AdobeStock_239258811.svg')]">
+    <div class="navbar flex justify-between items-center my-2 mx-5 flex-row py-2 px-4 ">
+
+        <div class="logos items-center ml-[18%] xl:ml-[10%] l:ml-[0px]"> 
```

Comment:
l:ml-[0px] is not correct you can use lg:ml-0

Reply:
sir actually l is defined by myself in the tailwind.config.js that has the functionality of Media Query